kmall.py

```diff
@@ -15,9 +14,55 @@
 import os
 import re
 import bz2
+import copy
 from pyproj import Proj
 from scipy import stats
 
+recs_categories = {'SKM': ['header.dgtime', 'sample.KMdefault.roll_deg', 'sample.KMdefault.pitch_deg',
```

Comment:
Sure, no problem there.  I was trying to avoid muddying up your code,  Wasn't sure how you'd feel about 60 lines of dictionary in your init.  I think that would bother some people...